modules/drivers/sparksql/src/metabase/driver/sparksql.clj

```diff
@@ -88,12 +88,19 @@
   (when s
     (s/replace s #"-" "_")))
 
+(defn- valid-describe-table-row [row]
```

Comment:
https://github.com/bbatsov/clojure-style-guide#pred-with-question-mark

Reply:
If you really want to make this function spicy you can rewrite as follows 🌶 :

```clj
(defn- valid-describe-table-row? [{:keys [col_name data_type]}]
  (every? (every-pred (complement str/blank?)
                      (complement #(str/starts-with? % "#")))
          [col_name data_type]))
```

*  Change name to `valid-describe-table-row?` as I previously suggested in comment above
*  Destructure `row` to get `:col_name` and `:data_type` out in the arglist
*  Use `every?` which checks if a predicate function is true of every item in a collection
*  Use `every-pred` to construct a predicate function that is true if all predicate functions passed to it are true
*  Use `complement` to return a predicate function that returns the complement (i.e., opposite boolean value) of the original predicate function

Reply:
Sorry I missed it renaming the function with the question mark.
Thank you for the cool tips on Clojure.